pkg/asset/cluster/tfvars.go

```diff
@@ -422,6 +424,8 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			Data:     data,
 		})
 	case baremetal.Name:
+		ironicCreds := &baremetalbootstrap.IronicCreds{}
+		parents.Get(ironicCreds)
```

Comment:
Get this asset at the top of the function, along with all of the other assets. This makes it easier to keep the dependencies used in `Generate` and the dependencies listed in `Dependencies` up to date with each other.

Reply:
My thought here was that since this calls `getrandom()` (and therefore blocks if sufficient entropy is not available), we should avoid generating it on platforms that don't need it. But I guess that applied more to previous versions of this PR where it would have happened right at startup if not worked around.

Reply:
At this point in the codepath, the asset must already exist. The `getrandom()` call would have happened already.